### PR TITLE
gui: bump rust version

### PR DESCRIPTION
**Pull request by edouardparis**

### CODE REVIEW

**Comment by darosior** [comment]
If we are going to do this we should bump the clippy version checked in CI to make sure we don't reintroduce any new warnings..

**Comment by darosior** [comment]
Obviously if you are only going to fix the lints for the GUI, only bump the version in the CI for the GUI, not the daemon

File: gui/Cargo.lock [hunk 1]
```diff
@@ -239,9 +239,9 @@ checksum = "9e1b586273c5702936fe7b7d6896644d8be71e6314cfe09d3167c95f712589e8"
 
 [[package]]
 name = "base64"
-version = "0.21.0"
+version = "0.21.6"
```
Comment (darosior) [inline gui/Cargo.lock:242]: This commit says it bumps the Liana version but it doesn't.

Comment (edouardparis) [reply gui/Cargo.lock:260]: Yes, my bad. I need to update it

**Comment by edouardparis** [comment]
Self-ACK 3196a8025ea9686cf76f407043b3ef2f85e0b4fa

**Pull request merged**